pages/about.html

```diff
@@ -1,12 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link rel="stylesheet" href="/stylesheets/about.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
+
+
 <body>
-  
+  <header class="navbar">
+    <section class="container">
+      <h2 class="logo">Tapasya <span>Khatri</span></h2>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="portfolio.html">Portfolio</a></li>
+          <!-- <li><a href="blog.html">Blog</a></li> -->
+        </ul>
+      </nav>
+    </section>
+  </header>
+
+  <main class="about-content">
+    <img src="/images/TapasyaProfilePic.jpg" alt="TapasyaProfilePic">
+    <section class="about-text">
+      <h2>About Me</h2>
+      <h5>Software Engineering <span>|</span> Product Design</h5>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)